functions/Export-DbaInstance.ps1

```diff
@@ -351,9 +355,23 @@ function Export-DbaInstance {
                 $fileCounter++
                 Write-Message -Level Verbose -Message "Exporting Policy Management"
                 Write-ProgressHelper -StepNumber ($stepCounter++) -Message "Exporting Policy Management"
-                $null = Get-DbaPbmCondition -SqlInstance $server | Export-DbaScript -FilePath "$exportPath\$fileCounter-policymanagement.sql" -Append:$Append -BatchSeparator $BatchSeparator -NoPrefix:$NoPrefix
-                $null = Get-DbaPbmObjectSet -SqlInstance $server | Export-DbaScript -FilePath "$exportPath\$fileCounter-policymanagement.sql" -Append:$Append -BatchSeparator $BatchSeparator -NoPrefix:$NoPrefix
-                $null = Get-DbaPbmPolicy -SqlInstance $server | Export-DbaScript -FilePath "$exportPath\$fileCounter-policymanagement.sql" -Append:$Append -BatchSeparator $BatchSeparator -NoPrefix:$NoPrefix
+
+                $outputFilePath = "$exportPath\$fileCounter-policymanagement.sql"
+                $scriptText = ""
+                $policyObjects = @()
+
+                # the policy objects are a different set of classes and are not compatible with the SMO object usage in Export-DbaScript
+                $policyObjects += Get-DbaPbmCondition   -SqlInstance $server
+                $policyObjects += Get-DbaPbmObjectSet   -SqlInstance $server
+                $policyObjects += Get-DbaPbmPolicy      -SqlInstance $server
```

Comment:
The tabbing out of the parameter spacing is not needed here. It just looks unnormal to me, that is used when you deal with syntax like PSCustomObjects but not parameters.